boot.php

```diff
@@ -83,6 +83,13 @@
 
 rex_extension::register('FE_OUTPUT', static function (rex_extension_point $ep) {
     if (true === rex_get('consent_manager_outputjs', 'bool', false)) {
+        if (!isset($_SESSION) || !isset($_SESSION['consent_manager'])) {
```

Comment:
@skerbis @TobiasKrais 
Vielleicht könnt Ihr das ja noch einmal verifizieren :)

Reply:
Hab mir gerade die Version von dir gezogen. Jetzt kommt der Cookie Dialog gar nicht mehr. Auch nicht, wenn ich auf den entsprechenden Button klicke. Der Event Listener ist nicht mehr an das Button gebunden.

Reply:
Korrektur: hab den Fehler mit dem Cookie Dialog gefunden. War in meiner Konfiguration. Änderungen sind getestet. Es funktioniert alles korrekt.